`inscribed_rect` test failing occasionally

## Describe the bug

A clear and concise description of what the bug is.

## Steps to reproduce

Run the CI enough time and eventually it fails.
```python
 tests/geometry/test_inscribed_rect.py:90: in <listcomp>
    Loop(*seg.discretize(byedges=True, ndiscr=50))
bluemira/geometry/_deprecated_loop.py:116: in __init__
    self._check_lengths()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <bluemira.geometry._deprecated_loop.Loop object at 0x7fa779f56ca0>

    def _check_lengths(self):
        """
        Checks if the input coordinates have appropriate lengths
    
        Raises
        ------
        GeometryError
            If the lengths are inappropriate
        """
        lengths = []
        for coord in [self.x, self.y, self.z]:
            if coord is not None:
                if hasattr(coord, "__len__"):
                    if len(coord) != 1:
                        lengths.append(len(coord))
    
        if len(lengths) < 2:
>           raise GeometryError("Insufficient Loop dimensions specified.")
E           bluemira.geometry.error.GeometryError: Insufficient Loop dimensions specified.
```

## Expected behaviour

A clear and concise description of what you expected to happen.

## Screenshots / Tracebacks

If applicable, add screenshots / tracebacks to help explain your problem.

## Environment (please complete the following information)

- OS: [e.g. Ubuntu 18.04]
- Version [e.g. 0.0.x]
- Branch (if developing) [e.g. develop]

## Additional context

Add any other context about the problem here.

--- a/tests/geometry/test_inscribed_rect.py
+++ b/tests/geometry/test_inscribed_rect.py
@@ -44,12 +44,14 @@ class TestInscribedRectangle:
         c_s.translate(i)
         complex_shape = boolean_cut(complex_shape, c_s)[0]
     # Convert back to Loop
-    complex_shape = Loop(*complex_shape.boundary[0].discretize(byedges=True, ndiscr=100))
+    complex_shape = Loop(
+        *complex_shape.boundary[0].discretize(byedges=True, ndiscr=100).xz
+    )
 
     shapes = [square, diamond, circle, complex_shape]
     convex = [True, True, True, False]
 
-    aspectratios = np.logspace(-1, 1, num=10)
+    aspectratios = np.logspace(-1, 1, num=5)
 
     @pytest.mark.parametrize("shape, convex", zip(shapes, convex))
     def test_inscribed_rectangle(self, shape, convex):
@@ -87,7 +89,7 @@ class TestInscribedRectangle:
                                 BluemiraFace(make_polygon(sq.xyz)), shape_face
                             )
                             tf = [
-                                Loop(*seg.discretize(byedges=True, ndiscr=50))
+                                Loop(*seg.discretize(byedges=True, ndiscr=50).xz)
                                 for seg in tf
                             ]
                         except ValueError:

